### PR TITLE
Merge alert component

### PR DESCRIPTION
**Pull request by piyushsh25**
Screenshot (176)

### CODE REVIEW

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    display: flex; 
+    align-items: center;
+    padding:var(--size5) var(--size20);
+    width: 100%;
+    background-color: var(--color-background);
+    font-size: var(--size10);
+    white-space: nowrap;
+    position: relative;;
+}
+.icon{
```
Comment (junaidshaikh-js) [inline components/alert/alert.css:15]: There should be a blank line between the CSS declaration box. Use code formatter to automate.

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+cursor: pointer;
```
Comment (junaidshaikh-js) [inline components/alert/alert.css:16]: Use consistent indentation over your whole file. Again use code formatter.

File: components/alert/alert.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="alert.css">
+    <title>Document</title>
```
Comment (junaidshaikh-js) [inline components/alert/alert.html:8]: The title should have a meaningful name.

File: components/alert/alert.css [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+height: var(--size14); 
+/* margin-left: auto; */
```
Comment (stutisk) [inline components/alert/alert.css]: no need to give comments

**Review by stutisk — COMMENTED** [review]
else your code looks good to me

**Comment by dubeyaditya29** [comment]
LGTM

**Pull request merged**